packages/expo-cli/src/commands/eject/Eject.js

```diff
@@ -227,6 +228,15 @@ from \`babel-preset-expo\` to \`babel-preset-react-native-stage-0/decorator-supp
     pkgJson.scripts.ios = 'react-native run-ios';
     pkgJson.scripts.android = 'react-native run-android';
 
+    const { sdkVersion } = exp;
+    const versions = await Versions.versionsAsync();
+    const reactNativeVersion = versions['sdkVersions'][sdkVersion]['facebookReactNativeVersion'];
+
+    if (semver.satisfies(sdkVersion, '31 - 32')) {
```

Comment:
@sergeichestakov is this only needed for SDKs 31 and 32 (and explicitly NOT for 33 and above)? If not would prefer something like `semver.gte(sdkVersion, '31')` to be more forward-compatible.